### PR TITLE
Better logging

### LINKED ISSUE #509: Enhance logging
## Description
We need to log a couple more stuff to trace how things pan out better, along with reconsidering the return part.

### PR DESCRIPTION
**Pull request by rakanalh**
# Description
Info level should provide minimal but easy to digest information. 

## Linked Issues
- Fixes #509

### CODE REVIEW

**Comment by eyusufatik** [comment]
good pr
will hold this for a few days tho
until we finalize the merge to the main branch.

**Comment by rakanalh** [comment]
> good pr will hold this for a few days tho until we finalize the merge to the main branch.

Sure, it's incomplete. What i plan to do is have the nodes output almost nothing but the block numbers with INFO level and decide on everything we log between debug and trace.

File: crates/sovereign-sdk/full-node/sov-prover-storage-manager/src/lib.rs
Comment (kpp) [inline crates/sovereign-sdk/full-node/sov-prover-storage-manager/src/lib.rs:110]: Keep this one with level=trace, remove err/ret

File: crates/sovereign-sdk/adapters/mock-da/src/service.rs
Comment (kpp) [inline crates/sovereign-sdk/adapters/mock-da/src/service.rs:87]: MockDA

File: crates/sequencer/src/sequencer.rs
Comment (kpp) [inline crates/sequencer/src/sequencer.rs:539]:
```suggestion
                    txs_count = txs.len(),
```

Comment (rakanalh) [reply crates/sequencer/src/sequencer.rs:539]: You would have to clone `txs` because it moved in the lines before which is more expensive than just getting the count before the move.

Comment (kpp) [reply crates/sequencer/src/sequencer.rs:539]: Oh, right!

**Review by kpp — APPROVED** [review]

File: crates/sovereign-sdk/full-node/sov-stf-runner/src/prover_service/parallel/mod.rs
Comment (eyusufatik) [inline crates/sovereign-sdk/full-node/sov-stf-runner/src/prover_service/parallel/mod.rs:76]: these are info imo

File: crates/sovereign-sdk/full-node/sov-stf-runner/src/prover_service/parallel/mod.rs
Comment (eyusufatik) [inline crates/sovereign-sdk/full-node/sov-stf-runner/src/prover_service/parallel/mod.rs:180]: keeping this info is helpful as well

File: crates/sovereign-sdk/full-node/sov-stf-runner/src/prover_service/parallel/prover.rs
Comment (eyusufatik) [inline crates/sovereign-sdk/full-node/sov-stf-runner/src/prover_service/parallel/prover.rs:152]: info

File: crates/sovereign-sdk/full-node/sov-stf-runner/src/runner.rs
Comment (eyusufatik) [inline crates/sovereign-sdk/full-node/sov-stf-runner/src/runner.rs:370]: info

File: crates/sovereign-sdk/full-node/sov-stf-runner/src/runner.rs
Comment (eyusufatik) [inline crates/sovereign-sdk/full-node/sov-stf-runner/src/runner.rs:586]: info + it's location is wrong can you fix?

File: crates/sovereign-sdk/full-node/sov-stf-runner/src/runner.rs
Comment (eyusufatik) [inline crates/sovereign-sdk/full-node/sov-stf-runner/src/runner.rs:867]: info

File: bin/citrea/src/lib.rs
Comment (eyusufatik) [inline bin/citrea/src/lib.rs:20]: i don't think it makes sense to have our main log initializer have a paremeter *just for* mock da tracing levels.

why not make a secondary init log function just for tests?

**Review by eyusufatik — CHANGES_REQUESTED** [review]

File: crates/sequencer/src/sequencer.rs
Comment (rakanalh) [inline crates/sequencer/src/sequencer.rs:539]: Use `txs_to_run`

File: bin/citrea/src/main.rs
Comment (eyusufatik) [inline bin/citrea/src/main.rs:69]: if default for v is 2

how do i set it for 0 ?

Comment (rakanalh) [reply bin/citrea/src/main.rs:75]: Added quiet arg

File: bin/citrea/tests/e2e/mod.rs
Comment (kpp) [inline bin/citrea/tests/e2e/mod.rs:2152]: ?

**Review by eyusufatik — APPROVED** [review]

**Pull request merged**
